__init__.py

```diff
@@ -293,31 +304,46 @@ def launch_librespot(self):
         if platform == 'mycroft_mark_1' and not path:
             path = 'librespot'
 
-        if (path and self.device_name and
-                'user' in self.settings and 'password' in self.settings):
-
-            # Disable librespot logging if not specifically requested
-            outs = None if 'librespot_log' in self.settings else DEVNULL
-
-            # TODO: Error message when provided username/password don't work
-            self.process = Popen([path, '-n', self.device_name,
-                                  '-u', self.settings['user'],
-                                  '-p', self.settings['password']],
-                                  stdout=outs, stderr=outs)
-
-            time.sleep(3)  # give libreSpot time to start-up
-            if self.process and self.process.poll() is not None:
-                # libreSpot shut down immediately.  Bad user/password?
-                if self.settings['user']:
-                    self.speak_dialog("FailedToStart")
-                self.process = None
-                self.librespot_starting = False
-                return
+        if not path:
```

Comment:
In many cases it's quite ok that librespot can't be started for example on a desktop install librespot isn't used at all.